pallets/space-ownership/tests/src/tests.rs

```diff
@@ -5,169 +5,182 @@
 // Full license is available at https://github.com/dappforce/subsocial-parachain/blob/main/LICENSE
 
 use frame_support::{assert_noop, assert_ok};
-use sp_runtime::traits::Zero;
 
-use pallet_ownership::Error as SpaceOwnershipError;
-use pallet_spaces::Error as SpacesError;
+use pallet_ownership::{EntityWithOwnership, Error as OwnershipError, Event as OwnershipEvent};
 
 use crate::{mock::*, tests_utils::*};
 
 #[test]
-fn transfer_space_ownership_should_work() {
-    ExtBuilder::build_with_space().execute_with(|| {
-        assert_ok!(_transfer_default_space_ownership()); // Transfer SpaceId 1 owned by ACCOUNT1 to ACCOUNT2
-
-        assert_eq!(SpaceOwnership::pending_space_owner(SPACE1).unwrap(), ACCOUNT2);
-    });
+fn transfer_ownership_works() {
```

Comment:
It has, but in the inner assert function. However, I created a separate test, because the assert function is reused in different tests, but if we have a separate test and it fails, we'll know, what exact problem do we have.